src/docs/admin/references/NodeSourcePolicyReference.adoc

```diff
@@ -286,7 +286,8 @@ tasks running on (default is false).
 
 The *Dynamic policy* is similar to the <<_scheduler_loading_policy>> in the sense that it can acquire/release nodes according to the
 scheduler load. The *Dynamic policy* is more advanced and gives a fuller range of configuration parameters which allow to:
- - filter the scheduler load to specific tasks which use the `NODESOURCENAME` <<_glossary_generic_information,generic information>>.
+
+ - filter the scheduler load to specific tasks which use the <<../user/ProActiveUserGuide.adoc#_node_source_generic_info,NODE_SOURCE>> or the (deprecated) <<../user/ProActiveUserGuide.adoc#_nodesourcename,NODESOURCENAME>> <<_glossary_generic_information,generic information>>.
```

Comment:
filter the scheduler load to **consider** specific tasks which use the ..
I deem the sentence is clearer like this